getinvolved.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: "Get Involved: Hello World"
+scripts:
+- "utils.js"
+- "sort.js"
+style: style.css
+---
+<div class="details-container">
+    <div class="details-item">
+
+        <!--Meet the Instructors-->
+        <div class="section">
+            <h2>Volunteer</h2>
+
+            <p><strong>What:</strong> Hello World, a summer camp that teaches middle school girls the basics of web
+                development.</p>
+
+            <p><strong>Who:</strong> You! We’re looking for volunteers who know (or are willing to learn) the basics of
+                HTML,
+                CSS, and Javascript. <strong>Programming experience is appreciated but NOT required -- we provide free
+                    training!</strong></p>
+
+            <p><strong>When:</strong> 2:00-4:00pm PST on Aug. 3-7 and Aug. 10-14. <strong>You don’t have to come every day;
+                    just days
+                    that work for you!</strong> Expect to do at least one hour of volunteer training beforehand, or more
+                if you
+                don’t know how to code.</p>
+
+            <p><strong>Benefits:</strong> Gain up to 25+ volunteer hours (or choose to do less), join a community of
+                programmers, and empower girls to
+                code. We’ll teach you invaluable coding and leadership skills for free, and you can gain loads of
+                volunteer
+                hours or a
+                position on your resume while you’re at it.
+            </p>
+            <p>
+                Contact us at <a href="mailto:{{site.contact.email}}">{{site.contact.email}}</a> for more info.
+            </p>
+        </div>
+        <a href="https://docs.google.com/forms/d/e/1FAIpQLSeapmj33WC0HLmJRsl2ICq3KTplADuCi2BM97jbV2GSQcP1TA/viewform?usp=sf_link" class="signup-wrap embedfallbackbutton">
+                <button class="action-button">Volunteer</button>
+            </a>
+            <iframe id="embeddedform" src="https://docs.google.com/forms/d/e/1FAIpQLSeapmj33WC0HLmJRsl2ICq3KTplADuCi2BM97jbV2GSQcP1TA/viewform?embedded=true" width="640" height="1443" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
```

Comment:
Do we want to keep the google form embedded? IMO i think we could probably change it back to a button that says like "volunteer here" or something, although this is open to discussion

Reply:
I think it would look cleaner to have a button instead.

Reply:
> Do we want to keep the google form embedded? IMO i think we could probably change it back to a button that says like "volunteer here" or something, although this is open to discussion

lets talk about that during standup today.

Reply:
i think selena mentioned that thi is supposed to be a super hacky/temporary thing so we can prob wait to fix it in the real version